faqs/galaxy/tools_help.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Where is the tool help?
+area: tools   
+description:  Finding tool support
+box_type: tip
+layout: faq    
+contributors: [jennaj, beachyesh]
+---
```

Comment:
I think that you need some blank lines here so that it renders correctly

Reply:
I added a blank line after the horizontal rule. That should suffice right?